xml/System/TimeSpan.xml

```diff
@@ -5139,9 +5139,9 @@ The value of this constant is 10.
 
 This property converts the value of this instance from ticks to microseconds.
              This number might include whole and fractional microseconds.
-            
-             The <xref:System.TimeSpan.TotalMicroseconds> property represents whole and fractional microseconds,
-             whereas the <xref:System.TimeSpan.Microseconds> property represents whole microseconds.
+
+The <xref:System.TimeSpan.TotalMicroseconds%2A> property represents whole and fractional microseconds,
```

Comment:
`%2A` is only needed if you want to link to all overloads of a method. It's not needed for properties.

Reply:
Yes, if the text is indented by 4 spaces in Markdown, it's automatically converted into a code block.